python_modules/libraries/dagster-k8s/dagster_k8s/client.py

```diff
@@ -755,42 +756,43 @@ def get_pod_debug_info(
             else {}
         )
 
-        for container in pod.spec.containers if (pod and pod.spec and pod.spec.containers) else []:
-            container_name = container.name
-            log_str = ""
+        if not skip_logs:
```

Comment:
suggestion: view diff without whitespace - only indentation changes for the lines below this one